packages/firestore/src/core/firestore_client.ts

```diff
@@ -507,22 +506,19 @@ export class FirestoreClient {
   }
 
   shutdown(): Promise<void> {
-    return this.asyncQueue.enqueue(async () => {
-      if (!this._clientShutdown) {
-        // PORTING NOTE: LocalStore does not need an explicit shutdown on web.
-        if (this.lruScheduler) {
-          this.lruScheduler.stop();
-        }
-        await this.remoteStore.shutdown();
-        await this.sharedClientState.shutdown();
-        await this.persistence.shutdown();
-
-        // `removeChangeListener` must be called after shutting down the
-        // RemoteStore as it will prevent the RemoteStore from retrieving
-        // auth tokens.
-        this.credentials.removeChangeListener();
-        this._clientShutdown = true;
+    return this.asyncQueue.enqueueAndInitilizeShutdown(async () => {
```

Comment:
`Initilize` => `Initialize`

FWIW, If you are using VS Code (if not, disregard), there's a "Code Spell Checker" extension from "Street Side Software" which is handy.

Reply:
Actually, `Initialize` => `Initiate` to match the other ports.